app/api/profiles/route.ts

```diff
@@ -21,18 +21,19 @@ export async function GET() {
 
 export async function POST(request: NextRequest) {
   try {
-    const userData: CreateProfilePayload = await request.json()
```

Comment:
**nitpick:** This is wrong domain - we should not create user in `profile`, but in `user`. This endpoint should be used only for creating profile